spec/fs_spec.lua

```diff
@@ -18,9 +22,485 @@ describe("Luarocks fs test #whitebox #w_fs", function()
       it("argument with special characters", function()
          assert.are.same(is_win and [["\\"%" \\\\" \\\\\\"]] or [['\% \\" \\\']], fs.Q([[\% \\" \\\]]))
       end)
+     
+   end)
+
+--testing fs_lua.current_dir()
```

Comment:
no need for these comments: the `describe` argument is enough.

Reply:
I will remove that.